src/plugins/memdump/memdump.cpp

```diff
@@ -537,6 +537,116 @@ static event_response_t terminate_process_hook_cb(drakvuf_t drakvuf, drakvuf_tra
     return VMI_EVENT_RESPONSE_NONE;
 }
 
+static void process_visitor(drakvuf_t drakvuf, addr_t eprocess, void* visitor_ctx)
+{
+    vmi_pid_t temp_pid;
+    pass_context* ctx = static_cast<pass_context*>(visitor_ctx);
+
+    if (!drakvuf_get_process_pid(drakvuf, eprocess, &temp_pid))
+    {
+        PRINT_DEBUG("[MEMDUMP] Failed to get process pid\n");
+        return;
+    }
+    if (temp_pid == *ctx->pid)
+    {
+        PRINT_DEBUG("[MEMDUMP] Found remote process base! Getting dtb..\n");
+        drakvuf_get_process_dtb(drakvuf, eprocess, ctx->dtb);
+        *ctx->process = eprocess;
+    }
+}
+
+static bool get_process_by_handle(drakvuf_t drakvuf, drakvuf_trap_info_t* info, uint64_t handle, addr_t* process, addr_t* dtb)
```

Comment:
Looks like this function may be useful at other places too, so move it to `libdrakvuf`?

Reply:
I guess so. I will do it as soon as possible.

Reply:
Done.